packages/manager/apps/pci-block-storage/src/pages/onboarding/OnBoarding.page.tsx

```diff
@@ -0,0 +1,185 @@
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import {
+  Card,
+  isDiscoveryProject,
+  OnboardingLayout,
+  PciDiscoveryBanner,
+  PublicCloudProject,
+} from '@ovhcloud/manager-components';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_TEXT_LEVEL,
+  OdsBreadcrumbAttributeItem,
+} from '@ovhcloud/ods-components';
+import { OsdsBreadcrumb, OsdsText } from '@ovhcloud/ods-components/react';
+import { useContext, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Outlet,
+  useNavigate,
+  useParams,
+  useRouteLoaderData,
+} from 'react-router-dom';
+import { GUIDES, IN_SUBSIDIARY } from './onboarding.constants';
+import OnBoardingGuard from '@/pages/onboarding/OnBoardingGuard';
+
+export default function OnBoardingPage() {
+  const { t } = useTranslation();
+  const { t: tOnBoarding } = useTranslation('onboarding');
+  const { projectId } = useParams();
+  const context = useContext(ShellContext);
+  const { navigation } = context.shell;
+  const { ovhSubsidiary } = context.environment.getUser();
+  const project = useRouteLoaderData('blocks') as PublicCloudProject;
+  const [urlProject, setUrlProject] = useState('');
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    navigation
+      .getURL('public-cloud', `#/pci/projects/${projectId}`, {})
+      .then((data) => {
+        setUrlProject(data as string);
+      });
+  }, [projectId, navigation]);
+
+  const breadcrumbItems: OdsBreadcrumbAttributeItem[] = [
+    {
+      href: urlProject,
+      label: project?.description,
+    },
+    {
+      label: t('pci_projects_project_storages_blocks_title'),
+    },
+  ];
+
+  const tileItems = [
+    {
+      id: 'create-attach-volume',
+      texts: {
+        title: tOnBoarding(
+          'pci_projects_project_storages_blocks_onboarding_guides_create-attach-volume_title',
+        ),
+        description: tOnBoarding(
+          'pci_projects_project_storages_blocks_onboarding_guides_increase-volume-size_description',
+        ),
+        category: tOnBoarding('onboarding_guide_title'),
+      },
+      href:
+        GUIDES['create-attach-volume'][
+          ovhSubsidiary === IN_SUBSIDIARY ? 'asiaLink' : 'defaultLink'
+        ],
+    },
+    {
+      id: 'attach_pci_floating_ip_to_instance',
+      texts: {
+        title: tOnBoarding(
+          'pci_projects_project_storages_blocks_onboarding_guides_configure-volume_title',
+        ),
+        description: tOnBoarding(
+          'pci_projects_project_storages_blocks_onboarding_guides_create-attach-volume_description',
+        ),
+        category: tOnBoarding('onboarding_guide_title'),
+      },
+      href:
+        GUIDES['configure-volume'][
+          ovhSubsidiary === IN_SUBSIDIARY ? 'asiaLink' : 'defaultLink'
+        ],
```

Comment:
**nitpick:** Instead of storing the link as `asia`, use `IN` as key and simplify the logic here.
```suggestion
        GUIDES['configure-volume'][ovhSubsidiary] || GUIDES['configure-volume'][default],
```